docs/native/src/faq/filetypes.rst

```diff
@@ -0,0 +1,16 @@
+What file types are supported in |Office iOS|?
```

Comment:
Actually, this may be documented elsewhere in Office support documentation. For example: https://support.office.com/en-us/article/Read-only-and-compatible-files-3c7ee711-27de-4cd5-8125-a6e1f5ed30c4?ui=en-US&rs=en-US&ad=US

I'd prefer linking to that documentation, since the file format support really isn't WOPI-related, and the apps may add additional file support in the future and the support docs will likely be updated at that point.